Dockerfile

```diff
@@ -22,6 +22,7 @@ COPY *.tar.gz .
 
 ## install dependencies not yet in qmongr-base-r
 RUN R -e "install.packages('yaml')"
+RUN R -e "remotes::install_github('rstudio/shiny')"
```

Comment:
Does not seem to work (on Travis at least). @areedv: do you have any recommendations how to solve this?